jupyterlab/staging/webpack.config.js

```diff
@@ -11,8 +11,8 @@ const Handlebars = require('handlebars');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 const webpack = require('webpack');
 const merge = require('webpack-merge').default;
-const BundleAnalyzerPlugin = require('webpack-bundle-analyzer')
-  .BundleAnalyzerPlugin;
+const BundleAnalyzerPlugin =
```

Comment:
Isn't this file copied by `update-core-mode`? If so maybe it should be ignored by prettier / ESLint?

Reply:
:+1: